### PR TITLE
Html changes

### PR DESCRIPTION
**Pull request by hajiler**
Adds detailed body in the form of paragraphs/lists to the heading skeleton.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -10,7 +10,24 @@
     <div id="content">
       <h1>Elijah's Portfolio</h1>
       <h4>Bio</h4>
+       <p> I was born and raised in Chicago, Illinois. I currently live in Logan Square,
+           and I study Computer Science at the Univerity of Illinois at Chicago. In my 
+           free time I enjoy biking,dancing, and reading comic books.
```
Comment (jessrenteria) [inline portfolio/src/main/webapp/index.html]: Nit: s/biking,dancing/biking, dancing/

Note: if you ever see us put "nit" in a code review, all it means is that we're nitpicking something not that important (like a small typo). Usually it's just the difference between correctness and polish. Putting another pair of eyes on a PR will almost always surface something like this, even on mine or Ali's :)

**Comment by hajiler** [comment]
I intended to create separate pull requests, 1 for finishing the written details of the page, and the other for adding an image folder containing a profile picture. But I added them both to same the branch and can't separate them now. So my question is should I be creating separate branches for each commit pull-request?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -10,7 +10,24 @@
...
+        <ul>
+            <li>Research: Utilized natural language processing and Naive Bayes classifiers 
+                to identify urgent and actionable requests for aid on Twitter in the event 
+                of a hurricane with 75% accuracy.
+            </li>
+            <li>School Project: Created UML diagrams to develop a game utilizing multi-threading 
+                and socket programming to develop a real-time, multi-client game along with a 
+                graphical user interface for both the server and the player.
+            </li>
+            <li>Personal Project: Trained a Text Generating Recusrive Neural Network on the entire
```
Comment (jessrenteria) [inline portfolio/src/main/webapp/index.html]: Nit: s/Recusrive/Recursive/

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -10,7 +10,24 @@
     <div id="content">
       <h1>Elijah's Portfolio</h1>
       <h4>Bio</h4>
+       <p> I was born and raised in Chicago, Illinois. I currently live in Logan Square,
+           and I study Computer Science at the Univerity of Illinois at Chicago. In my 
+           free time I enjoy biking,dancing, and reading comic books. 
+       </p>
       <h4>Projects</h4>
```
Comment (jessrenteria) [inline portfolio/src/main/webapp/index.html:19]: Consider adding links to these if you have them.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -10,7 +10,24 @@
...
+            <li>Research: Utilized natural language processing and Naive Bayes classifiers 
+                to identify urgent and actionable requests for aid on Twitter in the event 
+                of a hurricane with 75% accuracy.
+            </li>
+            <li>School Project: Created UML diagrams to develop a game utilizing multi-threading 
+                and socket programming to develop a real-time, multi-client game along with a 
+                graphical user interface for both the server and the player.
+            </li>
+            <li>Personal Project: Trained a Text Generating Recusrive Neural Network on the entire 
+                script of Avatar: The Last Air Bender to create character-specific dialogue.
```
Comment (jessrenteria) [inline portfolio/src/main/webapp/index.html]: I want to see some of the dialogue that sounds awesome!

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -10,7 +10,24 @@
     <div id="content">
       <h1>Elijah's Portfolio</h1>
       <h4>Bio</h4>
+       <p> I was born and raised in Chicago, Illinois. I currently live in Logan Square,
+           and I study Computer Science at the Univerity of Illinois at Chicago. In my 
+           free time I enjoy biking,dancing, and reading comic books. 
+       </p>
       <h4>Projects</h4>
+        <ul>
+            <li>Research: Utilized natural language processing and Naive Bayes classifiers
```
Comment (jessrenteria) [inline portfolio/src/main/webapp/index.html]: Indentations should be 2 spaces to conform to the [style guide](https://google.github.io/styleguide/htmlcssguide.html#Indentation).

This lets a reader easily distinguish between indentations and [line-wrapping](https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping).

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -10,7 +10,24 @@
     <div id="content">
       <h1>Elijah's Portfolio</h1>
       <h4>Bio</h4>
+       <p> I was born and raised in Chicago, Illinois. I currently live in Logan Square,
```
Comment (jessrenteria) [inline portfolio/src/main/webapp/index.html]: Nit: <p> isn't actually a child element of the headers above, so I don't think it should be indented at all.

Some [example code](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns) from the style guide shows it aligned with no indentation.

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -29,6 +29,13 @@ <h4>Projects</h4>
             </li>
         </ul>
       <h4>Organizations</h4>
+       <ul>
```
Comment (jessrenteria) [inline portfolio/src/main/webapp/index.html]: 2 space indent

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -29,6 +29,13 @@ <h4>Projects</h4>
...
+           <li> Society of Hispanic Professional Engineers</li>
```
Comment (jessrenteria) [inline portfolio/src/main/webapp/index.html]: 2 space indent.

Also, consider dropping the leading space on all of these.

**Review by jessrenteria — DISMISSED** [review]

**Comment by jessrenteria** [comment]
> I intended to create separate pull requests, 1 for finishing the written details of the page, and the other for adding an image folder containing a profile picture. But I added them both to same the branch and can't separate them now. So my question is should I be creating separate branches for each commit pull-request?

In general, I tend to prefer smaller, isolated changes. Here's a discussion on the benefits of doing so: https://google.github.io/eng-practices/review/developer/small-cls.html

Our internal code review tool has some pretty good support for doing this with separate branches that depend on each other, but it looks like git / GitHub [doesn't](https://stackoverflow.com/q/26619478). This is probably something we'll have to spend a bit of time to figure out. For now feel free to just add dependent commits to the same branch, and just keep the individual commits tiny, as that will still let us keep our code reviews focused.

If you have some totally independent commits, go ahead and make a new branch so you have can a separate PR.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (hajiler) [reply portfolio/src/main/webapp/index.html]: Sound good, thanks!

**Review by ahorjia — APPROVED** [review]

**Review by ahorjia — APPROVED** [review]

**Review by jessrenteria — APPROVED** [review]

**Pull request merged**